### DISCUSSION TITLE
Migrating  v12.0 partner_multi_relation_tabs to v13

### DISCUSSION THREAD

**Discussion by marcos-mendez** [OCA/OpenUpgrade#2981] [Q&A]
Dear colleagues, we have been trying for days to migrate a very simple module from v12 to v13 and consequently to v14 but we are totally stuck without being able to move forward.

We are talking about partner-contact/partner_multi_relation_tabs

We didn't find in the whole github or in the OCA pages what happened with this v13 module and therefore we don't know what to do.

Normally we would open an Issue but as here in this openupgrade repo there is the discussion I am opening a thread and marking @pedrobaeza  that did a great part of the migration. 

How should we procedure? Also what are the procedures to "COPY" the multi_relation_tabs from partner-contact v12? An error pops up saying that No Module Named **_odoo.osv.orm no_** what has to be done? Sorry for the ignorance but we are really stuck

**Comment by enieber** [comment]
maybe this PR is relationship with this problem: https://github.com/OCA/partner-contact/pull/936
